docs/getting-started/f44rm-how-it-works.mdx

```diff
@@ -51,6 +51,23 @@ If you haven't installed Dagger yet, we recommend you do it in order to consolid
 
 <Button label="Get Started" docPath="install" />
 
+export const CardGrid = ({ data }) => {
+  return (
+    <div className={styles.cardGrid}>
+      {data.map((x, index) => (
+        <Card
+          key={index}
+          label={x.label}
+          description={x.description}
+          relatedContent={x.relatedContent}
+          icon={x.imgFilename}
+          url={x.url}
+        />
+      ))}
+    </div>
+  );
+};
+
 export const data = {
```

Comment:
Do we import those data somewhere else ?

Reply:
It's [the same](https://docusaurus.io/docs/next/markdown-features/react#exporting-components) for a component and an object, we must export it, otherwise, Docusaurus throws an error.